code/modules/mob/new_player/new_player.dm

```diff
@@ -31,6 +33,12 @@
 	new_character = null
 	return ..()
 
+/mob/new_player/proc/reconsider_job_panel()
+	var/output = "<div align='center'>"
+	output += "<p>There is a lack of xenos players on this round, unbalanced rounds are unfun for everyone.</i></p>"
+	output += "<p><b>Are you sure you want to play as a marine?</b></p>"
+	output += "<br><p><a href='byond://?src=[REF(src)];lobby_choice=continue_join'>Yes</A> | <a href='byond://?src=[REF(src)];lobby_choice=reconsider'> No </A></p>"
+	return output
```

Comment:
ditto use tgui_alert() instead if possible

Reply:
TOO LATE YOU APPROVED I WONT DO NOTHING AH

Reply:
> TOO LATE YOU APPROVED I WONT DO NOTHING AH

SUFFER DO NOTHING NO MERGE